src/modules/app/constants/dimensions.css

```diff
@@ -1,6 +1,9 @@
 @custom-media --small-viewport (max-width: 750px);
+@custom-media --min-max-width screen and (min-width: 1100px);
```

Comment:
Not related to this task, but are there any official recommendations for the min-width values. If not then these are also magic values.

Reply:
Well, comes down to how `official` you want :D Theres no number in the html/css spec, but there are probably some suggested values for them in guides by google or other companies...
I think we should set them on how they best fit our design...

Reply:
Yeah I'm thinking more about good recommendations :D